backend/open_api_spec.yml

```diff
@@ -4,24 +4,72 @@ servers:
 paths:
   /curation/auth/key:
     post:
-      summary: Regenerate the curator's API key
+      summary: Generate the curator's API key
       security: 
         - cxguserCookie: [] 
-		get:
-      summary: Return the identifier of the curator's active API key
+      responses:
+        "201":
+          description: Created
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  key:
+                    type: string
+                  id:
+                    type: string
+        "401":
+          $ref: "#/components/responses/401"
+    get:
+      summary: Return the identifier of the curator's active API keys
       security: 
-        <API key-checking function>
-          #    delete:
-          #      summary: Delete the curator's API key
-          #      security: 
-          #        - cxguserCookie: [] 
-
+        - cxguserCookie: []
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: string
+        "401":
+          $ref: "#/components/responses/401"
+    delete:
+      summary: Delete the specified curator's API key
+      security:
+        - cxguserCookie: []
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                id:
+                  type: string
+      responses:
+        "202":
```

Comment:
had meant to ask why you're quoting the error codes like `"<code>"`

Reply:
Just following the spec https://swagger.io/docs/specification/describing-responses/

Reply:
interesting, fair enough